src/main/java/mate/academy/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy;
+
+public class DataProcessingException extends RuntimeException {
+    private String message;
+    private Throwable ex;
+
+    public DataProcessingException(String message, Throwable ex) {
+        this.message = message;
+        this.ex = ex;
+    }
+}
```

Comment:
Use `super` instead of creating fields, for example:

```java
public class DataProcessingException extends RuntimeException {
    public DataProcessingException(String message, Throwable ex) {
        super(message, ex);
    }

    public DataProcessingException(String message) {
        super(message);
    }
}
```